powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v3/PowerAuthUpgradeTest.java

```diff
@@ -1030,8 +1027,8 @@ void upgradeSignatureVerificationDuringUpgradeTest() throws Exception {
     }
 
     @Test
+    @Disabled
```

Comment:
I would rather always use this annotation with the reason
```suggestion
    @Disabled("Here explain why the test is disabled")
```